travis.sh

```diff
@@ -141,8 +142,10 @@ my_travis_wait 60 catkin build --no-status --summarize || exit 1
 # Source the new built workspace
 travis_run source install/setup.bash;
 
-# Only run tests on the current repo's packages
-TEST_PKGS=$(catkin_topological_order $CI_SOURCE_PATH --only-names)
+# Choose which packages to run tests on
+echo "Test blacklist: $TEST_BLACKLIST"
+TEST_PKGS=$(catkin_topological_order src --only-names | grep -Fvxf <(echo "$TEST_BLACKLIST" | tr ' ;,' '\n'))
```

Comment:
I noticed you removed $CI_SOURCE_PATH - this is the path to the target repo e.g. ``moveit_core`` vs all of moveit's repos. In the past I believe we've only run tests on the "current" repos packages - in the case of moveit_kinetic_cpp11 that was actually nothing since the repo was fake. But I guess that feature wasn't working since it was still running tests on all the packages. I don't understand this section of the CI, it was inherited from https://github.com/ros-planning/moveit_core/commit/ebeb18614ee2d12ab3304fb4aea39a7457c4766d#diff-354f30a63fb0907d4ad57269548329e3R66

I'd be happy to switch back to your PR (if you copy over the documentation change) if you could figure this section out better. I'm busy with the moveit merge right now